client/src/document/baseline-indicator.scss

```diff
@@ -0,0 +1,151 @@
+@use "../ui/vars" as *;
+
+$browsers: "chrome", "edge", "firefox", "safari";
+
+.baseline-indicator {
+  --baseline-bg: var(--baseline-unsupported-bg);
```

Comment:
🟠 Where is `--baseline-bg` used?

Reply:
A few lines below: `background: var(--baseline-bg);`